tests/test_files.py

```diff
@@ -156,25 +156,13 @@ def test_push_file_autostart(self):
         calls = [call(autostart, 'autostart'), call('%exclude ' + autostart, 'services')]
         self.fm.push_package_file.assert_has_calls(calls)
 
-    def test_push_file_extras(self):
-        """
-        Test push_file to extras package
-        """
-        self.fm.file_is_locale = MagicMock(return_value=False)
-        self.fm.push_package_file = MagicMock()
-        self.fm.extras.append('test')
-        self.fm.push_file('test', '')
-        calls = [call('test', 'extras')]
-        self.fm.push_package_file.assert_has_calls(calls)
-
-
     def test_push_file_custom_extras(self):
         """
         Test push_file to a custom extras package
         """
         self.fm.file_is_locale = MagicMock(return_value=False)
         self.fm.push_package_file = MagicMock()
-        self.fm.custom_extras = {'test-extras': {'files': ["test"]}}
+        self.fm.file_maps = {'test-extras': {'files': ["test"]}}
```

Comment:
Seems like an easily confused custom name since test-extras is very close to tests-extras which is special given the other parsing in this PR.

Reply:
(I know this wasn't your choice of name just maybe we should change it)